src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -422,6 +859,15 @@ public void binaryContentRedirectWantDigest(final String uri) {
                 "Fedora servers supporting \"redirect\" external content types MUST correctly respond to the " +
                         "\"Want-Digest\" header.",
                 "https://fedora.info/2018/06/25/spec/#redirect-and-proxy", ps);
+
+        final String location = createExternalBinary(uri, info.getId(), "redirect", null);
+
+        final String checksum = "md5;q=0.3,sha;q=1";
+        final Response wantDigestResponse = doGetUnverified(location, new Header("Want-Digest", checksum));
+
+        final Headers responseHeaders = wantDigestResponse.getHeaders();
+        assertTrue(responseHeaders.getValue(DIGEST).contains("md5") ||
```

Comment:
We may want to expand the list of accepted digest algorithms to include: sha256 and sha512.